src/components/TildeltVeileder.tsx

```diff
@@ -46,10 +47,10 @@ interface VeilederNavnProps {
 }
 
 function VeilederNavn({ tildeltVeilederident }: VeilederNavnProps) {
-  const veilederInfoQuery = useVeilederInfoQuery(tildeltVeilederident);
-  const veilederInfo = veilederInfoQuery.data;
+  const { isLoading, data: veilederInfo } =
+    useVeilederInfoQuery(tildeltVeilederident);
 
-  return veilederInfoQuery.isSuccess ? (
+  return !isLoading ? (
```

Comment:
Her var det en bug som gjorde at veileder-ident ikke ble vist dersom kallet for å hente veileder-navn feilet.